bundle/config/mutator/apply_transforms.go

```diff
@@ -0,0 +1,177 @@
+package mutator
+
+import (
+	"context"
+	"path"
+	"slices"
+	"sort"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/textutil"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/databricks/databricks-sdk-go/service/ml"
+)
+
+type applyTransforms struct{}
+
+// Apply all transforms, e.g. the prefix transform that
+// adds a prefix to all names of all resources.
+func ApplyTransforms() *applyTransforms {
+	return &applyTransforms{}
+}
+
+type Tag struct {
+	Key   string
+	Value string
+}
+
+func (m *applyTransforms) Name() string {
+	return "ApplyTransforms"
+}
+
+func (m *applyTransforms) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	diag := validatePauseStatus(b)
+	if diag != nil {
+		return diag
+	}
+
+	r := b.Config.Resources
+	t := b.Config.Transform
+	prefix := t.Prefix
+	tags := toTagArray(t.Tags)
+
+	// Jobs transforms: Prefix, Tags, JobsMaxConcurrentRuns, TriggerPauseStatus
+	for i := range r.Jobs {
```

Comment:
You can do:
```
for _, job := range r.Jobs
```

To work on the pointer and avoid indexing everywhere in the body.